docker-compose.dev.yml

```diff
@@ -22,7 +22,6 @@ services:
     environment:
       - DJANGO_SETTINGS_MODULE=config.settings.docker_dev
       - DJANGO_LOG_LEVEL
-      - EEEJ_LOAD_SMALL_TEST_DATASET=True
```

Comment:
Should the docker-compose.dev.yml include this env var?

Reply:
I think so because that is what we use for our GitHub regression testing, right? We want that to run faster.